Cython/Compiler/Visitor.py

```diff
@@ -844,6 +844,8 @@ def repr_of(self, node):
                 result += "(type=%s, name=\"%s\")" % (repr(node.type), node.name)
             elif isinstance(node, Nodes.DefNode):
                 result += "(name=\"%s\")" % node.name
+            elif isinstance(node, Nodes.CFuncDefNode):
+                result += "(name=\"%s\")" % node.declared_name()
```

Comment:
This is obviously pretty unrelated to the PR but it did help me debug it. I can remove it if needed though.